llvm/lib/Target/AIE/AIEPostPipeliner.cpp

```diff
@@ -306,12 +446,23 @@ bool PostPipeliner::scheduleFirstIteration() {
 
     scheduleNode(SU, Actual);
     Info[N].Scheduled = true;
-    LLVM_DEBUG(dbgs() << "Scoreboard\n"; Scoreboard.dumpFull(););
+    DEBUG_FULL(dbgs() << "Scoreboard\n"; Scoreboard.dumpFull(););
   }
-  LLVM_DEBUG(dbgs() << "==== First iteration scheduled ======\n");
+  LLVM_DEBUG(dbgs() << "==== First iteration scheduled by " << Strategy.name()
+                    << "====\n");
   return true;
 }
 
+namespace {
+void dumpEarliestChain(const std::vector<NodeInfo> &Info, int N) {
```

Comment:
Checking: I think we are safe because all those `dump` helpers seem to be in anonymous namespaces, but maybe adding a `[[nodiscard]]` would be safer?